docs/UserGuide.md

```diff
@@ -135,6 +135,58 @@ The suggested command will be automatically typed into the command bar.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Friends
+
+FriendDex allows you to store certain information about your friends, so you don't have to.
+The information below provides further description about your friends' details you can store.
+
+Further description on how to add/modify these fields are provided in the [Features](#features) section.
+
+### Name
+
+Your friend's name, so you don't have those embarrassing moments where you forget their name.
+
+This field is what uniquely identifies each friend in FriendDex.
```

Comment:
Can add in "Names are case-insensitive in FriendDex. This means that FriendDex interprets "John Doe" and "JoHn DoE" as the same name."